src/Stormpath/Authc/Api/ApiAuthenticationResult.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace Stormpath\Authc\Api;
+
+
+class ApiAuthenticationResult extends AuthenticatorResult
+{
+
+}
```

Comment:
Interesting. I was talking about the README specs, but I get what you are saying now. However, if those classes  don't have any implementation I don't see a reason to have them. Having said that, I think we could give them some use after taking a look at the Java SDK where each of the subclasses have some difference (usually getting a property that is unique for that type of result). See the interfaces on the Java SDK and their differences:

* [AuthenticationResult](https://github.com/stormpath/stormpath-sdk-java/blob/master/api/src/main/java/com/stormpath/sdk/authc/AuthenticationResult.java)
* [ApiAuthenticationResult](https://github.com/stormpath/stormpath-sdk-java/blob/master/api/src/main/java/com/stormpath/sdk/api/ApiAuthenticationResult.java)
* [AccessTokenResult](https://github.com/stormpath/stormpath-sdk-java/blob/master/api/src/main/java/com/stormpath/sdk/oauth/AccessTokenResult.java)
* [OauthAuthenticationResult](https://github.com/stormpath/stormpath-sdk-java/blob/master/api/src/main/java/com/stormpath/sdk/oauth/OauthAuthenticationResult.java)

So, we should do something similar or just use the one `AuthenticatorResult` that we have (which should have been called `AuthenticationResult`, but we'll roll with it).